atom/browser/browser_linux.cc

```diff
@@ -118,6 +123,12 @@ bool Browser::SetAsDefaultProtocolClient(const std::string& protocol,
 // |protocol|.
 bool Browser::IsDefaultProtocolClient(const std::string& protocol,
                                       mate::Arguments* args) {
+  std::string desktop_name;
+  if (!args->GetNext(&desktop_name)) {
```

Comment:
Should we check to see if it's empty?

Reply:
The empty desktop name does no harm. For IsDefaultProtocolClient, the final result will be false and SetAsDefaultProtocolClient would be no effect.